### ISSUE TITLE
[new lint] Empty first line in roxy examples

### ISSUE THREAD

**Issue by pawelru** [r-lib/lintr#2520]
Only after https://github.com/r-lib/lintr/issues/265

Empty lines following `#' @examples` roxygen tag are actually being rendered out. This is oftentimes confused with code-formatting within roxygen when blank lines got ignored.

Anti pattern (file `foo.R`):
```
#' @examples
#'
#' foo()
```

Which results into the following (file `foo.Rd`):
```
\examples{

foo()
```

Whereas it should be:
```
#' @examples
#' foo()
```

**Comment by IndrajeetPatil** [comment]
Maybe`{styler}` itself can get rid of such empty lines:

``` r
library(roxygen2)
styler::style_text("
#' Title
#'
#' @examples
#'
#' foo()
foo <- function() {}
"
)
#> #' Title
#> #'
#> #' @examples
#> #'
#> #' foo()
#> foo <- function() {}
```

<sup>Created on 2024-02-14 with [reprex v2.1.0](https://reprex.tidyverse.org)</sup>

That said, I'm hesitant to call this a lint, since some may actually prefer to have an empty like this after each tag to demarcate cleanly where one ends and the next one begins. But I guess this is something that can be customized if we do decide to lint this.

**Comment by MichaelChirico** [comment]
TBH I think this is for {styler}, if anything.
